src/AppInstallerCommonCore/UserSettings.cpp

```diff
@@ -518,6 +518,10 @@ namespace AppInstaller::Settings
                     m_type = UserSettingsType::Backup;
                     settingsRoot = settingsBackupJson.value();
                 }
+                else
+                {
+                    AICLI_LOG(Core, Warning, << "Failed loading settings files.");

```

Comment:
I was thinking more along the lines of the user visible portion (without opening the log file).  So a new:
```
m_warnings.emplace_back(StringResource::String::SettingsWarningWhoopsSettingsDidntLoad);
```
That would then at least warn the user "Hey, we couldn't load *any* settings, so you are getting the default behavior".